pallets/aggregate/src/lib.rs

```diff
@@ -513,9 +527,21 @@ pub mod pallet {
 
     /// Domains storage
     #[pallet::storage]
-    pub(crate) type Domains<T: Config> =
+    pub type Domains<T: Config> =
         StorageMap<Hasher = Blake2_128Concat, Key = u32, Value = Domain<T>>;
```

Comment:
Why relax permission here? If you need to see something in the domain storage implement a specific api: avoid to create dependencies on internal structs

Reply:
I don't need Domains in runtime anymore as we changed the payer account
undoing this